### ISSUE TITLE
Install fails if conda uses libmamba solver 

### ISSUE THREAD

**Issue by mlaugharn** [rsxdalv/tts-generation-webui#176]
I think this might be the same issue, and if so, should have the same fix: https://github.com/oobabooga/text-generation-webui/issues/2495

**Comment by mlaugharn** [comment]
Yes, just changed that line (adding --solver=classic) fixed the issue

**Comment by rsxdalv** [comment]
Thank you for the solution! I'll try to understand if this should be the
new default. I'm worried that some updates in the background might have
caused conda to fail.

**Comment by gepz** [comment]
Hi, I ran into the same problem 2 weeks ago. But adding "--solver=classic" didn't fix it on my WSL Ubuntu distro with mambaforge installed, because the default solver when installing packages will still be mamba (specified in ~\.condarc), which is not available in the local conda environment. (But it works now in my Windows environment, how though?).
I'm just a bit curious to know if anyone else had the same problem or if it was unique to my setup or a specific version of the repo.

To fix the solver, I ended up having to add a line to the startup script to create ./installer_files/conda/.condarc with the config:
```
solver: classic #!final
```
right after the miniconda install step in the script.
